tests/unit_tests/lcls_tools/common/data/test_archiver.py

```diff
@@ -645,211 +709,217 @@ def test_get_values_over_time_range_with_timedelta(self):
-                secs=1711144584,
-                val=6.509256903062449,
-                nanos=338140932,
+                secs=1712008584,
+                val=6.5091161400477375,
+                nanos=139994427,
                 severity=0,
                 status=0,
                 fields=None,
                 _timestamp=None,
+                meta=None,
             ),
             ArchiverValue(
-                secs=1711144585,
-                val=6.509256608192711,
-                nanos=452886090,
+                secs=1712008585,
+                val=6.5091153248503755,
+                nanos=252011512,
                 severity=0,
                 status=0,
                 fields=None,
                 _timestamp=None,
+                meta=None,
             ),
             ArchiverValue(
-                secs=1711144586,
-                val=6.509255910803729,
-                nanos=570835181,
-                severity=0,
-                status=0,
-                fields=None,
-                _timestamp=None,
-            ),
-            ArchiverValue(
-                secs=1711144587,
-                val=6.509256441099705,
-                nanos=686557679,
-                severity=0,
-                status=0,
-                fields=None,
-                _timestamp=None,
-            ),
-            ArchiverValue(
-                secs=1711144588,
-                val=6.5092569579434,
-                nanos=703651446,
-                severity=0,
-                status=0,
-                fields=None,
-                _timestamp=None,
-            ),
-            ArchiverValue(
-                secs=1711144589,
-                val=6.5092583877407435,
-                nanos=814986356,
+                secs=1712008589,
+                val=6.509116634443234,
+                nanos=706573951,
                 severity=0,
                 status=0,
                 fields=None,
                 _timestamp=None,
+                meta=None,
             ),
         ]
+        """
+        expected_result: DefaultDict[str, ArchiveDataHandler] = defaultdict(
+            ArchiveDataHandler
+        )
+        expected_result["ACCL:L0B:0110:DFBEST"] = ArchiveDataHandler(
+            value_list=dfbest_lst
+        )
+        expected_result["ACCL:L0B:0110:AACTMEAN"] = ArchiveDataHandler(
+            value_list=aactmean_lst
+        )
+        """
 
-        expected_result = {
-            "ACCL:L0B:0110:DFBEST": ArchiveDataHandler(value_list=dfbest_lst),
-            "ACCL:L0B:0110:AACTMEAN": ArchiveDataHandler(value_list=aactmean_lst),
-        }
+        pv_list = ["ACCL:L0B:0110:DFBEST", "ACCL:L0B:0110:AACTMEAN"]
+        value_lsts = [dfbest_lst, aactmean_lst]
+        expected_result: DefaultDict[str, ArchiveDataHandler] = defaultdict(
+            ArchiveDataHandler
+        )
+        for pv, value_lst in zip(pv_list, value_lsts):
+            expected_result[pv] = ArchiveDataHandler(value_lst)
```

Comment:
Is one of these better than the other? I have written two different ways to turn our `expected_result` into a DefaultDict. Both work, I'm wondering if one is preferred or easier to read?

Reply:
I personally like the first one better but again, developer's choice lol

Reply:
Sticking with the first option then!